pkg/model/summary.go

```diff
@@ -69,8 +72,55 @@ type Summary struct {
 	ScannedPaths []string `json:"paths"`
 }
 
+// PathParameters - structure wraps the required fields for temporary path translation
+type PathParameters struct {
+	ScannedPaths      []string
+	PathExtractionMap map[string]string
+}
+
+func getRelativePath(basePath, filePath string) string {
+	var rtn string
```

Comment:
I think we could rename `rtn` to be more clear... Maybe something like `resultPath`